javascripts/adaptive-placeholder.js

```diff
@@ -0,0 +1,20 @@
+app.adaptivePlaceholder = {
+
+  changeFilledState: function () {
+    var input = $(this);
+    var value = input.val();
+    
+    if(value != '' && value != undefined) {
```

Comment:
Would definitely recommend avoiding type coercion if at all possible. Still getting into the swing of the workflow, but additionally some linters would balk at the use of single quotes as well.